src/core/org/apache/jmeter/gui/util/JSyntaxTextArea.java

```diff
@@ -90,6 +95,9 @@ public JSyntaxTextArea(int rows, int cols, boolean disableUndo) {
         super.setLineWrap(LINE_WRAP);
         super.setWrapStyleWord(WRAP_STYLE_WORD);
         this.disableUndo = disableUndo;
+        if(USER_FONT_FAMILY!=null) {
+            setFont(new Font(USER_FONT_FAMILY, Font.PLAIN, USER_FONT_SIZE));
```

Comment:
@d0k1 your PR have a bug: when the jsyntaxtextarea.font.size property is defined and jsyntaxtextarea.font.family is *not* defined, the size is not change on the textarea.
Probably, you must define the default value to "JSyntaxTextArea.getDefaultFont().getName()" the USER_FONT_FAMILY constants, and remove this if condition.